jupyter_client/manager.py

```diff
@@ -67,6 +67,11 @@ def _kernel_spec_manager_default(self):
     def _kernel_spec_manager_changed(self):
         self._kernel_spec = None
 
+    shutdown_wait_time = Integer(
```

Comment:
I think a time in seconds should probably be `Float`, not `Integer` - someone may want to set it to e.g. 0.1 seconds.